packages/docs/site/docs/01-start-here/02-overview.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Overview
+slug: /overview
+---
+
+# Overview
+WordPress Playground is an online platform that allows you to experiment and learn about WordPress without affecting your live website. It's like a virtual sandbox where you can play around with different features, designs, and settings in a safe and controlled environment.
+
+## Here's how it works
+When you first start using WordPress Playground, you'll be provided with a separate space where you can create and customise your own WordPress website. This space is completely isolated from your actual website, so any changes you make here won't affect your live site.
```

Comment:
I'm curious – does playground.wordpress.net suggest that editing it could affect your website?

Reply:
Do you want me to remove the lines which have the "..affect.." word?

Reply:
Hm yes, that would work

Reply:
I have removed it.